microsoft-edge/webview2/how-to/debug-visual-studio-code.md

```diff
@@ -153,6 +172,36 @@ If you're debugging Office Add-ins, open the add-in source code in a separate in
 ```
 
 
+<!-- ====================================================================== -->
+## Debug WebView2 WinUI 2(UWP) Apps
+
+1. Install a WebView2 Runtime version past `106.0.1370.34`.
+
+1. Open the `Registry Editor` by clicking the **Windows Key** and searching for `Registry Editor`. Launch the `Registry Editor` application and select **Yes** to allow editing. Set the registry key “HKEY_CURRENT_USER\Software\Policies\Microsoft\Edge\WebView2\AdditionalBrowserArguments” equal to “--remote-debugging-pipe”.  To do this, follow the steps outlined in the [Debug running processes](#debug-running-processes) section.
```

Comment:
```suggestion
1. Open the `Registry Editor` by clicking the **Windows Key** and searching for **Registry Editor**. Open the **Registry Editor** application and select **Yes** to allow editing. 

1. Set the registry key `HKEY_CURRENT_USER\Software\Policies\Microsoft\Edge\WebView2\AdditionalBrowserArguments` equal to `--remote-debugging-pipe`.  To do this, follow the steps outlined in the [Debug running processes](#debug-running-processes) section above.
```
https://learn.microsoft.com/en-us/contribute/markdown-reference#apostrophes-and-quotation-marks